website/docs/r/postgresql_firewall_rule.html.markdown

```diff
@@ -0,0 +1,82 @@
+---
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_postgresql_firewall_rule"
+sidebar_current: "docs-azurerm-resource-postgresql-firewall-rule"
+description: |-
+  Creates a Firewall Rule for a PostgreSQL Server.
+---
+
+# azurerm\_postgresql\_firewall\_rule
+
+Creates a Firewall Rule for a PostgreSQL Server
+
+## Example Usage (Single IP Address)
+
+```hcl
+resource "azurerm_resource_group" "test" {
+  name     = "api-rg-pro"
+  location = "West Europe"
+}
+
+resource "azurerm_postgresql_server" "test" {
+  ...
+}
+
+resource "azurerm_postgresql_firewall_rule" "test" {
+  name                = "office"
+  resource_group_name = "${azurerm_resource_group.test.name}"
+  server_name         = "${azurerm_postgresql_server.test.name}"
+  start_ip_address    = "40.112.8.12"
+  end_ip_address      = "40.112.8.12"
+}
+```
+
+## Example Usage (IP Range)
+
+```hcl
+resource "azurerm_resource_group" "test" {
+  name     = "api-rg-pro"
+  location = "West Europe"
+}
+
+resource "azurerm_postgresql_server" "test" {
+  ...
```

Comment:
Totally minor, but I suggest using an indented `# ...` for "etc" placeholders, since it co-operates better with the syntax highlighting.

Reply:
Updated